### PR TITLE
Hpets16

### PR DESCRIPTION
**Pull request by ineiti**
Added docker-generation and the scripts needed to reproduce the hotpets16-demo

### CODE REVIEW

File: app/docker/cothority/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RUN echo 'export GOPATH=/home/dedis/go' >> ~/.profile; \
+   	echo 'export PATH=$PATH:/usr/lib/go-1.6/bin:~/bin:$GOPATH/bin' >> ~/.profile; \
+   	echo 'export GOPATH=/home/dedis/go' >> ~/.bashrc; \
+    echo 'export PATH=$PATH:/usr/lib/go-1.6/bin:~/bin:$GOPATH/bin' >> ~/.bashrc; \
+	. ~/.profile;  \
+	mkdir go; \
+	go get -v github.com/dedis/cosi; \
+	go get -v github.com/dedis/cothority
+
+EXPOSE 2000-2010
```
Comment (nikkolasg) [inline app/docker/cothority/Dockerfile]: 10 ports ?

File: app/docker/hpets16/clean_containers [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (nikkolasg) [inline app/docker/hpets16/clean_containers]: For all the bash scripts, it's better to have the `.sh` extensions :)

**Comment by nikkolasg** [comment]
Well except the minor things, I'm OK, you can even do it yourself because it goes to your branch anyway ...

**Comment by ineiti** [comment]
yes, if `identity_follower` doesn't go in before Friday, I'd prefer having a 'nice' branch-name ;)

**Pull request merged**